### DISCUSSION TITLE
Access to gist.

### DISCUSSION THREAD

**Discussion by Ttiki** [community/community#5921] [Mobile] [answered]
Hello.

I know I'm not suggesting the best feature this application needs, but I think it could be cool to get access to our gist from the application. 
I understand works needs to be done elsewhere, and a small feature as this one is not a necessity, but still, I post this in hope to get this feature added in the future. 🤞

Sincerly,
Ttiki.

**Comment by amrehman** [comment, marked as answer]
Thanks a lot for your feedback!

I've added your note to our internal issue where we are tracking adding gists. If you have any other feedback or feature requests, please keep them coming!

**Comment by AlexHedley** [comment]
#5332
#5614
#6466

**Reply by Wladefant** [reply]
#5332 is the oldest one and therefore the origin

**Reply by samfry13-13** [reply]
All
